### ISSUE TITLE
Anchor - add accessibility information on ds site

### ISSUE THREAD

**Issue by jcfilben** [grommet/hpe-design-system#4507]
Using the work done in this ticket: https://github.com/grommet/hpe-design-system/issues/4382. Create a Figma draft of the accessibility information that should be added to the Anchor component page. From there devs can add it to the site.

Deliverables:
- [x] determine which WCAG rules apply to the Anchor component
- [x] evaluate which rules are passing or failing (may need to test with automated testing in storybook, manual keyboard testing and manual screenreader testing)
- [x] use the template created in https://github.com/grommet/hpe-design-system/issues/4382 to create a section in the Anchor page for accessibility

**Comment by luketa8** [comment]
[Anchor A11y MVP](https://www.figma.com/design/0xPCwEV9SgNUriNjdQxKwk/%237447-Anchor----A11y-update?node-id=2-8733&t=XYGBF1gExv8sv2Rn-4) is Ready for Review. 

@halocline @jcfilben @SOjaHPE

**Comment by jcfilben** [comment]
Thanks @luketa8 added some comments

**Comment by luketa8** [comment]
Updated [Mockup](https://www.figma.com/design/0xPCwEV9SgNUriNjdQxKwk/%237447-Anchor----A11y-update?node-id=33-326&t=acgDQECkD4g8Fs8n-4).

**Comment by jcfilben** [comment]
Added additional comments

**Comment by jcfilben** [comment]
Looks good, thanks Lucas!
